helm/cas-ciip-portal/templates/mailhog-route.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.mailhog.enable }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: {{ template "cas-ciip-portal.fullname" . }}-mailhog
+  labels:
+{{ include "cas-ciip-portal.labels" . | indent 4 }}
+
+spec:
+  host: cas-mailhog-{{ .Values.route.host }}
+  port:
+    targetPort: 8025
```

Comment:
you can use the port name from the service (`http`) here instead of the port number. 
see https://github.com/codecentric/helm-charts/blob/bd58a27e4dfd4ed6c9f13bfbe2acb4968a1d4da1/charts/mailhog/templates/service.yaml#L31